src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -122,13 +122,13 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Target Name="_CollectTargetFrameworkForTelemetry" AfterTargets="_CheckForUnsupportedTargetFramework">
     <ItemGroup>
-      <TFTelemetry Include="TargetFrameworkVersion" Value="$([MSBuild]::Escape('$(TargetFrameworkMoniker)'))" />
-      <TFTelemetry Include="RuntimeIdentifier" Value="$(RuntimeIdentifier)" />
+      <TFTelemetry Include="TargetFrameworkVersion" Value="$([MSBuild]::Escape('$(TargetFrameworkMoniker)'))" Hash="true" />
+      <TFTelemetry Include="RuntimeIdentifier" Value="$(RuntimeIdentifier)" Hash="true" />
       <TFTelemetry Include="SelfContained" Value="$(SelfContained)" />
-      <TFTelemetry Include="UseApphost" Value="$(UseApphost)" />
+      <TFTelemetry Include="UseAppHost" Value="$(UseAppHost)" />
```

Comment:
Here's the typo - every other usage in the SDK is looking for `UseAppHost`, not `UseApphost`.